src/components/question-wrapper.tsx

```diff
@@ -219,7 +219,8 @@ export default class QuestionWrapper extends React.Component<IProps, IState> {
       return false;
     }
     // There's an explanation or at least one choice marked as correct.
-    return correctExplanation || question.choices.filter((c: any) => c.is_correct === true).length > 0;
+    const choices = question.choices ? question.choices : JSON.parse(question.authored_state).choices;
+    return correctExplanation || choices.filter((c: any) => c.is_correct === true || c.correct === true).length > 0;
```

Comment:
I'm not sure if the use of `=== true` is necessary, but I decided to follow the pattern already being used here.

Reply:
I don't think the teacher-edition-plugin should look at the authored_state. This author_state might change. And we might also have some interactives that we want to show the correct tab, which aren't multiple choice interactives.  So I think if the question object doesn't have `question.choices` then the logic should just be based on correctExplanation.  In other words if it is multiple choice interactive it shouldn't be treated any differently than another interactive.

So you could just return `[]` for the choices, 
or change the logic to `correctExplanation || (question.choices && question.choices.filter... )

I realize that the `isMCQuestion` being called above breaks what I'm saying. Perhaps we can fix that function at the same time.  It is also used to decide whether to call findInputInWrappedQuestion. And in findInputInWrappedQuestion it is assuming the question is a built in question time (not an interactive). So I'd suggest changing `isMCQuestion` to be `isBuiltInMCQuestion`. Then `isBuiltInMCQuestion` would be used by showCorrectTab instead of checking `question.choices`.

Reply:
@scytacki Thanks. I understand the first part, but I'm a little unclear on the suggested changes around `isMCQuestion`. Sorry.

`isMCQuestion` checks whether the question is a native LARA MC question or a Library Interactive MC question. You're saying it should only check for native LARA MC questions? So the function could be cut down to simply `return wrappedEmbeddableContext.type === LARA_MULTIPLE_CHOICE`?

Is it right that we then wouldn't show the Correct and Distractor tabs for Library Interactive MC questions that have a correct answer specified? It seems like that's something people would want as a default feature for MC question wrappers, but maybe it's not?